icicle-compiler/test/Icicle/Test/Sea/Seaworthy.hs

```diff
@@ -34,7 +35,7 @@ namer = A.namerText (flip Var 0)
 --   Like prop_psv, but without psv.
 prop_seaworthy wt
  = testIO
- $ do let Just a = asAttributeName "eval"
+ $ do let a = [inputid|default:eval|]
```

Comment:
the old line means `[input|eval]` literally (with an empty namespace). If we are disallowing empty namespaces and use `default` as the default namespace, the new line does mean the same thing. It might be better to keep the semantics exactly the same as the old line just to make sure we are testing the same thing.